build/tasks/release.js

```diff
@@ -49,6 +49,7 @@ gulp.task('bundle', function () {
   var bundles = [
     'app/cart/cart.component',
     'app/checkout/checkout.component',
+    'app/thank-you/thank-you.component',
```

Comment:
I like the naming conventions here https://angular.io/styleguide#!#naming but even I haven't been consistent with them. What do you think? The 3 of us should be on the same page but it doesn't matter to me whether that's camelCase of kebab-case for filenames. It looks like about half our filenames have dashes and the other half don't...